schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+id INT GENERATED ALWAYS AS IDENTITY,
+name string,
```

Comment:
- [OPTIONAL] It will be nice to use  `varchar` instead of `string`. Varchars gives the flexibility to set a character limit which is really good right?